### PR TITLE
Build and deploy site on Netlify

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Merge? That way we can already play around on Netlify. The page will be available here (until we add a custom domain): https://mixxx-website.netlify.app/

**Comment by Holzhaus** [comment]
Deploy Preview is here:
https://app.netlify.com/sites/mixxx-website/deploys/5ef361dc3d9a5400074ee29e
https://5ef361dc3d9a5400074ee29e--mixxx-website.netlify.app/

**Comment by ywwg** [comment]
Can you add information to a Readme or other documentation explaining what the status of netlify is?  or link to a google doc with mixxx.org access if it's secret information.  We should make this so anyone with website admin privileges can understand what's going on

**Comment by Holzhaus** [comment]
> Can you add information to a Readme or other documentation explaining what the status of netlify is? or link to a google doc with mixxx.org access if it's secret information. We should make this so anyone with website admin privileges can understand what's going on

https://github.com/mixxxdj/mixxx/wiki/Server-Migration#website

File: templates/footer.html [hunk 1]
```diff
@@ -54,7 +54,8 @@ <h2 class="docs-header">About</h2>
 
   <div style="text-align:center">
     &copy; 2001-2020 Mixxx Development Team<br>
-    Logo by Paul Bloch
+    Logo by Paul Bloch<br>
+    <a href="https://www.netlify.com/" target="_blank">This site is powered by Netlify.</a>
```
Comment (rryan) [inline templates/footer.html:58]: Is that a requirement of netlify?

Comment (rryan) [reply templates/footer.html:58]: Nevermind, just read the commit message.

**Pull request merged**